docs/concepts/ipld.md

```diff
@@ -11,4 +11,53 @@ related:
 
 # The InterPlanetary Linked Data (IPLD) model
 
-<ContentStatus />
+The Interplanetary Linked Data (IPLD) model is a single namespace for all hash-inspired protocols. This allows for interoperable data formats. For example, you can reference a Git commit to a Bitcoin transaction.
+
+As mentioned in the [specifications](https://github.com/ipld/specs), the goal of IPLD is to enable decentralized data-structures that are universally addressable and linkable, which in turn will enable more decentralized applications. These data structures allow us to do for data what URLs and links did for HTML web pages.
+
+IPFS, Bitcoin, and other applications use hash-linked data structures ([Merkle trees](https://docs-beta.ipfs.io/concepts/merkle-dag/)) to verify data integrity. However, each protocol has a very different format. IPLD's goal is to provide a data layer definition that allows the linking of different content-addressed data types.
+
+## Where does IPLD fit in?
+
+IPLD acts as a layer between networking protocols and higher-level applications. This enables developers to build decentralized applications without having to worry about lower-level changes. Both upper- and lower-layer technologies can grow independently, while IPLD acts as a middle layer that resolves the two. This is known as "thin waist" or "narrow waist" protocol.
+
+![A narrow-waist diagram showing where IPLD fits into the internet ecosystem.](https://gateway.ipfs.io/ipfs/QmXgrfpCcSFfXnXqSz6G3V9E21pTZdBmVdsBQCkr86kHXP)
+
+## The IPLD layer model
+
+The IPLD specifications are divided into [layers](https://github.com/ipld/specs#ipld-layer-model):
+
+0. **Block layer:** The block layer contains the content identifier (CID), which is used to self-describe the multi-base-prefix, version, content type, and content address. This takes the guessing out of encoding/decoding.
+1. **Data model layer:** This layer describes what is representable in IPLD (lists, booleans, maps, etc). The kinds of data available are listed in the [`ipld/specs` repository](https://github.com/ipld/specs/blob/master/data-model-layer/data-model.md#kinds).
+1. **Schema layer:** The schema layer uses the data model layer to build more complex layouts.
```

Comment:
It is not clear (if you are not deep into IPLD) what "complex layouts" is supposed to mean. Also the Schema layer is quite a bit more. For more information about it see https://specs.ipld.io/schemas/.

Reply:
:+1: to adding that link.

"complex layouts" is pretty vague, but for a high level overview I think maybe it's fine; I don't have other strong suggestions for how to phrase it, anyway.